### PR TITLE
4553 move aggregation move operation to async task

### PR DESCRIPTION
**Pull request by hyi**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4598](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4598)

**Comment by hyi** [comment]
@sblack-usu This PR is ready for review now that all tests have passed. @pkdash Feel free to take a look as well if you have time.

File: theme/static/js/hs_file_browser.js [hunk 1]
```diff
@@ -620,8 +622,10 @@ function paste(destPath) {
 
     // Wait for the asynchronous call to finish to get new folder structure
     $.when.apply($, calls).done(function () {
-        refreshFileBrowser();
-        clearSourcePaths();
+        if (!is_async_celery_task) {
```
Comment (sblack-usu) [inline theme/static/js/hs_file_browser.js:625]: I'm confused about how this works.  The comments a few lines up says `// Wait for the asynchronous call to finish to get new folder structure` but the folder structure is only refreshed when it is not an async task.

Comment (hyi) [reply theme/static/js/hs_file_browser.js:625]: Yeah, the asynchronous call in the comment meant the asynchronous called pushed to the calls stack in JavaScript which is different from what is_async_celery_task variable means. Will update the comment to make this clear.

**Review by sblack-usu — APPROVED** [review]

**Pull request merged**